src/pages/iou/request/step/IOURequestStepConfirmation.js

```diff
@@ -183,9 +199,25 @@ function IOURequestStepConfirmation({
                 transaction.merchant,
                 transaction.billable,
                 TransactionUtils.getValidWaypoints(transaction.comment.waypoints, true),
+                policy,
+                policyTags,
+                policyCategories,
             );
         },
-        [report, transaction],
```

Comment:
Why the change from a single `transaction`, to several `transaction.property`?

Reply:
That was an auto-fix from prettier/the linter, I didn't intentionally make this change.

That said, this is probably more accurate as to whether the hook should re-run, as it's going to be looking at values more directly instead of the whole object (which can pretty easily change identity even if all the values are equivalent, depending on the data source's implementation).

Reply:
It looks like when I used the auto-fix function to add `policy`, etc, to the dependency array, the auto-fix also updated the array to match everything it expected, including being more specific about the args. Like I said before, using `transaction` here isn't technically wrong, it's just probably less good.

That said I just pushed an update to remove the unnecessary change.